web/index.php

```diff
@@ -30,9 +30,14 @@
 // Instantiate a Slim app
 $app = new Slim(array(
     'log.enabled' => true,
-    'templates.path' => '../templates'
+    'templates.path' => __DIR__.'/../templates'
```

Comment:
Nope. If I deploy using the Heroku button and leave the `SLIM_MODE` set to `production`, I still get a 500 error for the /view endpoint ... unless I add this `__DIR__.` code.